src/components/article/Article.vue

```diff
@@ -32,23 +32,18 @@
                                 >
                                     <VsHeading
                                         level="2"
+                                        headingStyle="heading-xl"
                                         class="mb-8 mb-lg-9"
                                         :class="businessSupport ? null : 'text-center'"
                                     >
                                         <span :id="anchorLink ? anchorLink : ''">
                                             {{ title }}
                                         </span>
-                                    </VsHeading>
 
-                                    <VsRichTextWrapper
```

Comment:
Can this intro potentially include things other than a heading? If it can be a full para this seems like a loss of functionality (including the Business Support stuff)

Reply:
This was a design change to use the subheading instead of lead text I think with idea that it could still be a paragraph. The whole subheading thing needs re-worked. 

I can change it back to lead text for now and speak to the design team about this potentially being a long paragraph or maybe we can update it when we re-work the article if its going to require CMS changes.

Reply:
Okay cool, no that makes sense if it was an intentional removal of that functionality. When we come to merge in v4 we can resolve that change in Support Hub as we would any other breaking chang